test/tornado_tests/test_motor_core.py

```diff
@@ -169,10 +141,14 @@ async def test_gridout_attrs(self):
             'stream_to_handler'
         ]).union(motor_only)
 
+        gridin_only = set([
+            'md5', 'readlines', 'truncate', 'flush', 'fileno', 'closed', 'writelines',
+            'isatty', 'writable'])
```

Comment:
Can you open a new ticket to add support for these to motor? Or does it not make sense to implement IOBase in async world? Is there a std lib asyncio version of IOBase?

Reply:
These methods are unsupported in `pymongo` as [well](https://github.com/mongodb/mongo-python-driver/blob/c760f900f2e4109a247c2ffc8ad3549362007772/gridfs/grid_file.py#L680)